docs_source_files/content/getting_started/_index.nl.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Aan de slag"
+chapter: true
+weight: 1
+---
+
+# Aan de slag
+
+Hieronder vind je enkele links die je aan de slag helpen
+indien dit je eerste ervaring met Selenium is.
+
+* [Korte uitleg]({{< ref "/getting_started/quick.md" >}})
```

Comment:
I think this one needs to point to the file with the language suffix. `/getting_started/quick.nl.md`

Other ones are failing due the same reason.

Reply:
Fixed in new push